config/__init__.py

```diff
@@ -32,6 +32,11 @@ class Config:
         'AWS_TEAMS_TABLE': 'aws_teams_tablename',
         'AWS_PROJECTS_TABLE': 'aws_projects_tablename',
         'AWS_REGION': 'aws_region',
+        'AWS_LOCAL': 'aws_local',
+    }
+    OPTIONALS = {
+        'TESTING': 'False',
+        'AWS_LOCAL': 'False',

```

Comment:
Should be good to remove `TESTING` completely.

As for `AWS_LOCAL`, it should be a boolean instead of a string.